user/serializers.py

```diff
@@ -1 +1,29 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        read_only_fields = ("id", "is_staff")
+        extra_kwargs = {
+            "password": {"write_only": True,
+                         "min_length": 5}
+        }
+
+    def create(self, validated_data):
+        """create user with encrypted password."""
+        return get_user_model().objects.create_user(**validated_data)
```

Comment:
The `create` method uses `create_user` which is a common method in custom user models. Ensure that your user model has a `create_user` method implemented. If not, you may need to adjust this method to handle user creation appropriately.